api/app/models/concerns/with_markdown.rb

```diff
@@ -7,11 +7,11 @@ def render_simple_markdown(input)
       filter_html: true,
       no_images: true,
       no_links: true,
-      no_styles: true
+      no_styles: true,
+      hard_wrap: true
     }
     renderer = Redcarpet::Render::HTML.new(options)
     redcarpet = Redcarpet::Markdown.new(renderer)
-    out = redcarpet.render(input)
-    Regexp.new(%r{\A<p>(.*)<\/p>\Z}).match(out)[1]
+    redcarpet.render(input).strip
```

Comment:
It looks like descriptions can be multi-paragraph, so I changed this to return all the HTML.